src/Sulu/Bundle/ContactBundle/Import/Import.php

```diff
@@ -516,6 +528,53 @@ protected function processCsvLoop($filename, $function, $flushOnEveryRow = false
     }
 
     /**
+     * checks if data row contains a value that is defined as exclude criteria (for a specific column)
```

Comment:
Capital C `Check if data`

and should also leave an empty line before the first `@param`

````php
/**
 * Foo bar foo
 * 
 * @param ...
 */
````